src/renderer/qssWidget/js/qssVolumeWidget.js

```diff
@@ -108,6 +122,16 @@
         }
     });
 
+    /**
+     * Set the actual volume value in the case the volume is changed through the Windows itself
+     * @param {Component} that - The `gpii.psp.widgets.volume` instance.
+     * @param {Number} value - The value of the setting.
+     */
+    gpii.qssWidget.volume.loadActualValue = function (that, value) {
+        that.model.setting.value = value;
```

Comment:
Any assignment into the model needs to be done via the ChangeApplier

Reply:
Done, the change from the applier is on the next line. This was like a "backup", just in case, but I tested it without it and seems to work just fine now. Maybe it wasn't necessary after all ;-)